config

```diff
@@ -102,7 +102,7 @@ if [ "$ngx_wasm_cargo_lib_name" = ngx_wasm_rs ]; then
         fi
     fi
 
-    if echo -n "$ngx_wasm_runtime_libs\n$NGX_LD_OPT" | grep -q "ngx-wasm-rs"; then
```

Comment:
This greps for the path `ngx-wasm-rs` as it is specified as a path, not a library name. Please revert.

Reply:
The path name doesn't necessarily contain `ngx-wasm-rs` (for example the rust rule automatically export the file in `lib` under build dependency https://github.com/Kong/kong-ee/pull/6346/files#diff-5c8ca1cd9c76393e97326f7857dd2be6cd741552bab8e4413dbcad5b652a94ccR29). Well, it's probably not reliable to either test the path or the module name, as it may fasly match some other part in NGX_LD_OPT; how do you feel?

Reply:
> The path name doesn't necessarily contain `ngx-wasm-rs`

It does when using the local development environment, hence why keeping it that way. For Bazel there will have to be another way of detecting this.